src/sqlacodegen/cli.py

```diff
@@ -40,6 +52,13 @@ def main() -> None:
     )
     parser.add_argument("--noviews", action="store_true", help="ignore views")
     parser.add_argument("--outfile", help="file to write output to (default: stdout)")
+    parser.add_argument(
+        "--conv",
+        nargs="*",
+        help='constraint naming conventions in "key=template" format \
+e.g., --conv "pk=pk_%%(table_name)s" "uq=uq_%%(table_name)s_%%(column_0_name)s"',
```

Comment:
I added the option to add the naming convention via command line on top of config file (which hasn't been added yet)

I was thinking, for each config option, we can go like this:
- Check if it is available as command line arg
- If not, check if it is available in config file
- If not, fall back to default value

Reply:
How would users know what to enter here, based on this?

Reply:
Well we assume users interested in using this option are already familiar with SQLAlchemy's constraint naming convention feature from https://docs.sqlalchemy.org/en/14/core/constraints.html#configuring-constraint-naming-conventions or https://alembic.sqlalchemy.org/en/latest/naming.html. In this case, the `help` string above would enough in my opinion.

We can also add more details in README:
- Link to SQLAlchemy/Alembic docs above
- Maybe also basic info about constraint naming convention and its usage in SQLAlchemy, basically summarizing the docs above
- Things not supported in `sqlacodegen`:
  - Using custom function as naming convention token
  - Using the constraint object name as key instead of mnemonic (e.g., `PrimaryKeyConstraint` cannot be used as key, must use `pk`)